packages/data-stores/src/queries/use-jetpack-search-ai.ts

```diff
@@ -0,0 +1,32 @@
+import { useQuery } from 'react-query';
+import wpcomRequest from 'wpcom-proxy-request';
+
+type AIResponseURL = {
+	url: string;
+	title: string;
+};
+
+export type JetpackSearchAIResult = {
+	response: string;
+	step: string;
+	urls: AIResponseURL[];
+	terms: string[];
+};
+
+export function useJetpackSearchAIQuery( siteId: number | string, query: string, stopAt: string ) {
+	return useQuery< JetpackSearchAIResult >(
+		[ query ],
```

Comment:
We should prefix this with something uniquish, it's highly unlikely, but if someone for examples searches for ["happychat-available"](https://github.com/Automattic/wp-calypso/blob/fix/proper-rq-config/packages/happychat-connection/src/use-happychat-available.ts#L59) an incorrect value will be cached and Happychat auth will break.